cli/lib/util.js

```diff
@@ -175,8 +177,57 @@ const dequote = (str) => {
   return str
 }
 
+const parseOpts = (opts) => {
+  opts = _.pick(
+    opts,
+    'project',
+    'spec',
+    'reporter',
+    'reporterOptions',
+    'path',
+    'destination',
+    'port',
+    'env',
+    'cypressVersion',
+    'config',
+    'record',
+    'key',
+    'configFile',
+    'browser',
+    'detached',
+    'headed',
+    'global',
+    'dev',
+    'force',
+    'exit',
+    'cachePath',
+    'cacheList',
+    'cacheClear',
+    'parallel',
+    'group',
+    'ciBuildId'
+  )
+
+  if (opts.exit) {
+    opts = _.omit(opts, 'exit')
+  }
+
+  // some options might be quoted - which leads to unexpected results
+  // remove double quotes from certain options
+  const removeQuotes = {
+    group: dequote,
+    ciBuildId: dequote,
+  }
+  const cleanOpts = R.evolve(removeQuotes, opts)
```

Comment:
is there a reason not to apply this treatment to all of the CLI args? it's a little weird that we treat quotes specially for these 2 but not others